suricataparser/parse.py

```diff
@@ -91,3 +91,17 @@ def parse_file(path):
                 rules.append(rule)
             buffer = ""
     return rules
+
+
+def parse_rules(rules_object):
```

Comment:
Please, write unit test for this function

Reply:
Added unit test for new function.